src/components/NewLocationPage/LocationName/LocationName.style.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+import { materialSMBreakpoint } from 'assets/theme/sizes';
+
+export const RegionContainer = styled.div`
+  @media (max-width: ${materialSMBreakpoint}) {
+    text-align: center;
+  }
+`;
+
+export const RegionNameText = styled.div`
```

Comment:
we want to preserve the hierarchy of the page for accessibility purposes (and i think SEO purposes?). in this case, that would mean using the `h1` element for the location name (rather than a `div`). from an accessibility standpoint, this lets assistive technologies know that the location name is the main header of the page